ocs_ci/templates/ocp-deployment/install-config-ibm_cloud-ipi.yaml.j2

```diff
@@ -25,7 +25,7 @@ networking:
     hostPrefix: {{ cluster_host_prefix | default(23) }}
   machineNetwork:
   - cidr: 10.0.0.0/16
-  networkType: OpenShiftSDN
+  networkType: {{ network_type | default('OVNKubernetes') }}
```

Comment:
why default is not OpenshiftSDN?

Reply:
starting 4.12 OVNKubernetes is default networkType